.github/workflows/tests.yml

```diff
@@ -1,26 +1,26 @@
-name: Tests
+name: CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
not sure to understand why it works not and not before ?

Reply:
um github actions need to be manually enabled on the `actions` tab page, wasn't on previously. as for the `on: [push]`, think that's sufficient, don't need trigger on pull_request too

Reply:
ah ok makes sense now :D